run_tests.sh

```diff
@@ -0,0 +1,3 @@
+ninja executables_tests && \
```

Comment:
why we can't use cmake with ENABLE_TESTS?

Reply:
Not sure, what do you mean here?...

Reply:
Those tests are designed to be a cmake target. I am not changing this behaviour in this PR, it was like this before

Reply:
before we don't use separate script for run tests

Reply:
Yes, right. I could put it back into flake file. Konstantin says, it's a good practice to have it in a separate script, so you can just call it from nix develop, i.e. it's reusable